lib/cli.rb

```diff
@@ -38,7 +39,7 @@ def create_parser(sub_parser)
         end
       end
 
-      # rubocop:disable Metrics/MethodLength
+      # rubocop:disable Metrics/AbcSize,Metrics/MethodLength
```

Comment:
Perhaps the code assigning the default values could be extracted into another method though we don't have to try so hard to avoid rule disablements considering the nature of this method.

Reply:
Anyway, we will disable AbsSize for the next option.